### ISSUE TITLE
Swift 2.0 + XCode 7b4: Only classes that inherit from NSObject can be declared @objc

### ISSUE THREAD

**Issue by tomaszwojcik** [Quick/Quick#353]
I'm running into error from topic when trying to run tests (fresh project, running the sample test generated by Xcode).

My Podfile:
```
# Uncomment this line to define a global platform for your project
platform :ios, '9.0'
use_frameworks!

pod 'Alamofire', :git => 'https://github.com/Alamofire/Alamofire.git', :branch => 'swift-2.0'

def testing_pods
  pod 'Quick', '0.5.0'
  pod 'Nimble', '2.0.0-rc.1'
end

target 'CocoaPodsConfigurationIssueTests' do
  testing_pods
end

target 'CocoaPodsConfigurationIssueUITests' do
  testing_pods
end
```

**Comment by dpassage** [comment]
I hit this too; looks like the support has been checked in but not released yet. I got things working by going to the head source like this:

````
def testing_pods
    pod 'Quick', :git => 'https://github.com/Quick/Quick', :branch => 'swift-2.0'
    pod 'Nimble', :git => 'https://github.com/Quick/Nimble.git', :branch => 'swift-2.0'
end
````

**Comment by tomaszwojcik** [comment]
Thanks a lot for this workaround! <3

I've already reinstalled Xcode, dev tools, various ruby versions with various CocoaPods versions supporting Swift, with no success.

**Comment by deemadden** [comment]
This fixed the above problem for me as well, but uncovered a new one.  When I run my tests, I get the following warnings for both Nimble and Quick:

ld: warning: directory not found for option '-F/Applications/Xcode-beta.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator9.0.sdk/Developer/Library/Frameworks'

I tried setting ENABLE_BITCODE to No, but that didn't do it.  Any ideas?

**Comment by jeffh** [comment]
Yes, the current releases of Quick & Nimble aren't compatible with Xcode 7b4 yet. The warning about the missing directory can be safely ignored. It seems to be implicitly added when importing XCTest that I haven't had time to look into.

**Comment by jeffh** [comment]
I've released newer versions of Quick (0.5.1) & Nimble (2.0.0-rc2) which should work for Xcode 7b4. LMK if you have more issues.

**Comment by deemadden** [comment]
Confirmed the original issue is gone in this release.  Thanks Jeff!
